perf(docs-infra): improve app configuration

alan-agius4 opened the pull request:
- Remove trailing slash redirect (e.g., /overview no longer redirects to /overview/).
- Update static file matching: Resolves an issue where hashed files were being cached for up to 1 hour.

alan-agius4 commented:
@josephperrott, from my understand the firebase configuration of the `main` branch is used for the previous, is there a way to test config updates?

josephperrott commented on adev/firebase.json:
```diff
@@ -18,7 +19,7 @@
       },
       {
-        "source": "*.[0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f][0-9a-f].+(css|js)",
+        "source": "/**/*-[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z].+(css|js|png|jpeg|svg|jpg)",
```
Should this be `A-z` or do we only want to match capital letters?

alan-agius4 replied:
We want to match capitals.

josephperrott reviewed (approved):
LGTM

Just one question to confirm

alan-agius4 committed:
perf(docs-infra): improve app firebase configuration

- Remove trailing slash redirect (e.g., /overview no longer redirects to /overview/).
- Update static file matching: Resolves an issue where hashed files were being cached for up to 1 hour.

AndrewKushnir commented:
This PR was merged into the repository by commit 1f1fff3a4f9e6297c3cecc39299a84690c3ddb5c.

The changes were merged into the following branches: main, 19.0.x
